packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -178,6 +178,12 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
         value={textValue}
         onChange={onChange}
         onKeyDown={onTextFieldKeyDown}
+        onFocus={(e) => {
+          // Fix for setting the cursor to the correct position when multiline is true
+          // This approach should be reviewed during migration to FluentUI v9
+          e.currentTarget.value = '';
```

Comment:
Why do you have to set this to an empty string first?

Reply:
Because otherwise the value change will be ignored as `e.currentTarget.value` is already equal to `textValue` and cursor position (`selectionStart`&`selectionEnd`) won't be reset